bindings/meson.build

```diff
@@ -0,0 +1,9 @@
+build_rust = not get_option('rs').disabled()
+if build_rust
+	if not build_gir
+		error('cannot build "rust" without "introspection"')
+	else
+		subdir('rust')
```

Comment:
Can I ask for consistent 2 spaces indentation in all `meson.build` files, please?